frontend/src/routes/Search/Details/KubevirtPluginWrapper.tsx

```diff
@@ -2,18 +2,50 @@
 import React, { useContext, createContext } from 'react'
 import { PluginContext } from '../../../lib/PluginContext'
 import { KubevirtPluginData } from '../../../plugin-extensions/extensions/KubevirtContext'
+import { ResourceSearchLink } from './DetailsOverviewPage'
+import { K8sGroupVersionKind, ResourceIcon } from '@openshift-console/dynamic-plugin-sdk'
+import { useRecoilValue, useSharedAtoms } from '../../../shared-recoil'
+import classNames from 'classnames'
 
 // Define a default context
 const DefaultKubevirtPluginContext = createContext<KubevirtPluginData>({} as KubevirtPluginData)
 
-const ResourceLink = () => <div>Hello!!</div>
+const getResourceLinkOverride = (clusterName: string, inline?: boolean, truncate?: boolean, className?: string) => {
```

Comment:
What are the inline, truncate, and className parameters for? It looks to me like they are not used in your code.